.github/workflows/build.yml

```diff
@@ -152,6 +202,9 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
+      
+      - name: Fix VCS Linter Issue
```

Comment:
What is VCS? What is the issue?

Reply:
Go Linter tries to check some "build stamping" against some information from the VCS, but fails reading this information.